sap/adt/objects.py

```
@@ -283,6 +283,9 @@ def __init__(self, package=None, description=None, language=None,
     def package(self):
         """ABAP development package (DEVC)"""
 
+        #if self._package_reference is not None:
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)